parent/pom.xml

```diff
@@ -1320,7 +1321,7 @@
             <dependency>
                 <groupId>org.osgi</groupId>
                 <artifactId>org.osgi.compendium</artifactId>
```

Comment:
We should better use the updated coordinates (osgi.cmpn).

Reply:
done